music21/romanText/tsvConverter.py

```diff
@@ -410,6 +407,7 @@ def __init__(self):
         super().__init__()
         self.mn = None
         self.mn_onset = None
+        self.volta = None
```

Comment:
mentioning this because it's an addition, but it's the same for all of these.  It would be better if the default for these was not None but '' because this sets the typing of the usage later.  We're heading towards > 90% typing in mypy so it would be very helpful if these could be typed and if the typing was not  `str | None` but just `str` which requires a '' default.

Reply:
I notice for instance, that you're not doing "and thisChord.volta is not None" but instead just "and thisChord.volta" so setting the default to an empty string won't cause a problem.

Reply:
OK added some type hints in a new commit. Also initialized to the empty string where that didn't cause any hassle.